nerdlets/root/function-picker.js

```diff
@@ -21,35 +21,17 @@ export default class FunctionPicker extends React.PureComponent {
 
     if(!attribute || isCount) return <div/>
 
-    return <div className="function-picker-radio-container">
-      <div className="function-picker-radio-container-child">
-        {FUNCTIONS.map(f => {
-          const checked = fn == f.fn
-          return (
-            <Radio
-              key={f.fn}
-              onClick={() => setFunction(f.fn)}
-              checked={checked}
-              label={f.label}
-              className="function-picker-radio"
-              />
-          )
-        })}
-      </div>
-      <div className="function-picker-radio-container-child">
-        {FUNCTIONS2.map(f => {
-          const checked = fn == f.fn
-          return (
-            <Radio
-              key={f.fn}
-              onClick={() => setFunction(f.fn)}
-              checked={checked}
-              label={f.label}
-              className="function-picker-radio"
-              />
-          )
-        })}
+    const options = FUNCTIONS
+    return (
+      <div className="react-select-input-group">
+        <label>Function</label>
+        <Select
+          options={options}
+          value={{ value: fn, label: 'label' }} //TODO: get label to update based on state
```

Comment:
Hey, Josh. Thanks for the updates to the PR! I pulled down the changes from your forked repo and believe I know exactly what the problem is. One is here:

You can completely remove the `value={}` prop from the `<Select />` component. It's not needed for this to work and I believe it's preventing some updating functionality.

---

I'll leave a separate comment on the other.